apps/common/views/404.html

```diff
@@ -0,0 +1,12 @@
+{{<layout}}
+
+{{$header}}
+  {{title}}
+{{/header}}
+
+{{$content}}
+  <p>{{description}}</p>
+  <a href="/" class="button" role="button">Start again</a>
```

Comment:
I noticed the other day that this path `href="/"` needs to use `{{startLink}}`, it's a local created in the error middleware